examples/basic_charts/src/main.rs

```diff
@@ -697,49 +796,77 @@ fn basic_sankey_diagram() {
     plot.add_trace(trace);
     plot.set_layout(layout);
 
-    plot.show();
+    if show {
+        plot.show();
+    }
+    plot
 }
+// ANCHOR_END: basic_sankey_diagram
 
-fn table_chart() {
+// ANCHOR: table_chart
+fn table_chart(show: bool) -> Plot {
     let trace = Table::new(
         Header::new(vec![String::from("col1"), String::from("col2")]),
         Cells::new(vec![vec![1, 2], vec![2, 3]]),
     );
     let mut plot = Plot::new();
     plot.add_trace(trace);
-    plot.show();
+
+    if show {
+        plot.show();
+    }
+    plot
+}
+// ANCHOR_END: table_chart
+
+fn write_example_to_html(plot: Plot, name: &str) {
+    std::fs::create_dir_all("./out").unwrap();
+    let html = plot.to_inline_html(Some(name));
+    std::fs::write(format!("./out/{}.html", name), html).unwrap();
 }
 
 fn main() {
-    // Uncomment any of these lines to display the example.
+    // Change false to true on any of these lines to display the example.
 
     // Scatter Plots
-    // simple_scatter_plot();
-    // line_and_scatter_plots();
-    // bubble_scatter_plots();
-    // polar_scatter_plot();
-    // data_labels_hover();
-    // data_labels_on_the_plot();
-    // colored_and_styled_scatter_plot();
-    // large_data_sets();
-    // categories_scatter_chart();
+    write_example_to_html(simple_scatter_plot(false), "simple_scatter_plot");
```

Comment:
I wish there was a nicer way we could achieve this, but I think it is a small price to pay for the benefit of having the examples are compiled and baked in the book.